hieradata/clients/controller.sponsorship.ci.jenkins.io.yaml

```diff
@@ -392,6 +392,44 @@ profile::jenkinscontroller::jcasc:
           credentialsId: "jenkinsvmagents-userpass"
           usePrivateIP: true
           spot: true
+        - name: "ubuntu-22-inbound-amd64-jdk17"
+          description: "Ubuntu 22.04 LTS"
+          imageDefinition: jenkins-agent-ubuntu-22.04-amd64
+          os: "ubuntu"
+          os_version: "22.04"
+          launcher: "inbound"
+          location: "East US 2"
+          instanceType: Standard_D4ads_v5 # 4 vCPUS / 16 Gb / Max 150 Gb local storage
+          ephemeralOSDisk: true
+          architecture: amd64
+          labels:
+            - ubuntu-22-amd64-maven-17
+          javaHome: /opt/jdk-17 # Test override of the default JDK for builds
+          agentJavaBin: /opt/jdk-17 # Test override of the default JDK for builds
```

Comment:
```suggestion
```

This line must not be added: it specifies the "runtime JDK" (e.g. the one used to start the `agent.jar` process) and must be the same as the controller JDK runtime.

It is different than the JDK provided as default on CLI (as a build tool).

The agentJavaBin is specified globally once for all in https://github.com/jenkins-infra/jenkins-infra/blob/23392b6c47a954e9b60001b62e3d356ffa5119d1/hieradata/common.yaml#L217 (it can be specified at agent template level but we do not want it).